From 12e55eea61b2d8faa17aa0412cf8f69e72bbd857 Mon Sep 17 00:00:00 2001
From: Wallace Wang <r59996@freescale.com>
Date: Mon, 28 Jul 2008 17:43:40 +0800
Subject: [PATCH] ENGR00082738 Mono playback has loud white noise in left channel

Mono playback has loud white noise in left channel
Duplicate the right channel to Left channel since codec only support
stereo audio

Signed-off-by: Wallace Wang <r59996@freescale.com>
---
 sound/soc/codecs/wm8350.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/sound/soc/codecs/wm8350.c b/sound/soc/codecs/wm8350.c
index f73f480..7313dd0 100644
--- a/sound/soc/codecs/wm8350.c
+++ b/sound/soc/codecs/wm8350.c
@@ -1283,7 +1283,7 @@ static int wm8350_dapm_event(struct snd_soc_codec *codec, int event)
 
 static const struct snd_soc_pcm_stream wm8350_hifi_dai_playback = {
 	.stream_name	= "Playback",
-	.channels_min	= 1,
+	.channels_min	= 2,
 	.channels_max	= 2,
 	.rates		= WM8350_RATES,
 	.formats	= WM8350_FORMATS,
-- 
1.5.4.4

